packages/app/src/app/pages/Sandbox/Editor/Content/Preview/DevTools/Console.js

```diff
@@ -1,15 +1,36 @@
 import React from 'react';
 import Console from 'app/components/Preview/DevTools/Console';
 
+import ClearIcon from 'react-icons/lib/md/clear-all';
+import VerticalAlign from 'app/src/app/components/Preview/Navigator/VerticalAlign';
+import HorizontalAlign from 'app/src/app/components/Preview/Navigator/HorizontalAlign';
+
+import { dispatch } from 'codesandbox-api';
+
 import Navigator from './Navigator';
 
-export default ({ alignRight, alignBottom }) => (
-  <div style={{ height: '100%' }}>
-    <Navigator
-      alignRight={alignRight}
-      alignBottom={alignBottom}
-      title="Console"
-    />
-    <Console.Content />
-  </div>
-);
+export default ({ alignRight, alignBottom }) => {
+  const actions = [
```

Comment:
Much cleaner, nice!

Reply:
I was thinking we could maybe reuse the action exported by `Console.js`? These ones: https://github.com/CompuIves/codesandbox-client/blob/master/packages/app/src/app/components/Preview/DevTools/Console/index.js#L169. Maybe a simple concat.

Reply:
Yes makes sense.

Reply:
I didn't noticed this console module was already imported, much cleaner indeed :)

Reply:
PR updated 😉